lib/domodule.js

```diff
@@ -2,6 +2,9 @@
 
 import parentModule from '../lib/getParentModule';
 import attrObj from 'attrobj';
+import * as domassist from 'domassist';
```

Comment:
do we know if rollup supports wilcard?  we might need to put the names of the stuff we use so we can generate a smaller bundle